python/ray/rllib/setup-rllib-dev.py

```diff
@@ -11,28 +11,38 @@
 
 import ray
 
-if __name__ == "__main__":
-    rllib_home = os.path.abspath(os.path.join(ray.__file__, "../rllib"))
-    local_home = os.path.abspath(os.path.dirname(__file__))
-    assert os.path.isdir(rllib_home), rllib_home
+
+def do_link(package):
```

Comment:
the changes in this file are unrelated, right?

Reply:
This is for dev purposes (and are relatively harmless to include in this PR IMO)